packages/flutter_tools/lib/src/commands/assemble.dart

```diff
@@ -310,6 +310,14 @@ class AssembleCommand extends FlutterCommand {
         "Try re-running 'flutter build ios' or the appropriate build command."
       );
     }
+    if (deferredTargets.isNotEmpty) {
+      // Record to analytics whether DeferredComponents is being used.
```

Comment:
Since we don't send an event when deferred components is _not_ used:

```suggestion
      // Record to analytics that DeferredComponents is being used.
```